src/umpire/strategy/AllocationAdvisor.cpp

```diff
@@ -47,7 +47,7 @@ AllocationAdvisor::AllocationAdvisor(
       m_allocator);
 
 #if defined(UMPIRE_ENABLE_CUDA)
-  if (accessing_allocator.getPlatform() == Platform::cpu) {
+  if (accessing_allocator.getPlatform() == Platform::host) {
```

Comment:
@davidbeckingsale, you might want to add this to release notes.  I noticed that it broke at least one of the builds I was testing against that was using the old Platform::cpu.